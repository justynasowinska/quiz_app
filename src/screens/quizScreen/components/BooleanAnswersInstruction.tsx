import * as React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import { Icon } from 'react-native-elements';

import { THEME_COLORS } from '../../../config/colors';

export const BooleanAnswersInstruction = () => {
    return (
        <View style={styles.container}>
            <Text style={styles.answer}>FALSE</Text>
            <Icon
                name="arrow-long-left"
                type="entypo"
                size={20}
                iconStyle={styles.arrowIconLeft}
            />
            <Icon
                name="gesture-swipe-horizontal"
                type="material-community"
                size={40}
                color={THEME_COLORS.appTextColor}
                iconStyle={styles.iconStyle}
            />
            <Icon
                name="arrow-long-right"
                type="entypo"
                size={20}
                iconStyle={styles.arrowIconRight}
            />
            <Text style={styles.answer}>TRUE</Text>
        </View>
    );
};

const styles = StyleSheet.create({
    container: {
        flexDirection: 'row',
        alignItems: 'center',
        marginBottom: 40,
        marginTop: -40
    },
    answer: {
        fontSize: 20,
        color: THEME_COLORS.appTextColor,
        fontWeight: 'bold'
    },
    iconStyle: {
        marginLeft: 15,
        marginRight: 15
    },
    arrowIconLeft: {
        marginLeft: 15
    },
    arrowIconRight: {
        marginRight: 15
    }
});