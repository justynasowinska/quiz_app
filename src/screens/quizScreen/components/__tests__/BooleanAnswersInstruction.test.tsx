import * as React from 'react';
import { shallow } from 'enzyme';

import { BooleanAnswersInstruction } from '../BooleanAnswersInstruction';

describe('Quiz Screen BooleanAnswersInstruction Component', () => {
    it('matches to the snapshot', () => {
        const component = shallow(<BooleanAnswersInstruction />);

        expect(component).toMatchSnapshot();
    });
});
