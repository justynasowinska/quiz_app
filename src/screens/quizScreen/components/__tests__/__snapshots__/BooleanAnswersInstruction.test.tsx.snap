// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Screen BooleanAnswersInstruction Component matches to the snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "marginBottom": 40,
      "marginTop": -40,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#474D54",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    FALSE
  </Text>
  <Themed.Icon
    iconStyle={
      Object {
        "marginLeft": 15,
      }
    }
    name="arrow-long-left"
    size={20}
    type="entypo"
  />
  <Themed.Icon
    color="#474D54"
    iconStyle={
      Object {
        "marginLeft": 15,
        "marginRight": 15,
      }
    }
    name="gesture-swipe-horizontal"
    size={40}
    type="material-community"
  />
  <Themed.Icon
    iconStyle={
      Object {
        "marginRight": 15,
      }
    }
    name="arrow-long-right"
    size={20}
    type="entypo"
  />
  <Text
    style={
      Object {
        "color": "#474D54",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    TRUE
  </Text>
</View>
`;
